hpx/parallel/util/loop.hpp

```diff
@@ -48,6 +50,7 @@ namespace hpx { namespace parallel { namespace util
         };
     }
 
+
```

Comment:
All of the code related to prefetching should go into a separate file, e.g. `hpx/parallel/util/prefecthing.hpp`